Script/Packages/AtomicNET/Package.json

```diff
@@ -4,5 +4,5 @@
   "namespace" : "Atomic",
   "dependencies" : ["Script/Packages/Atomic"],
   "modules" : ["NETCore", "NETScript"],
-  "platforms" : ["WINDOWS", "MACOSX"]
+  "platforms" : ["WINDOWS", "MACOSX","LINUX"]
```

Comment:
This could be a little iffy, we'll see what happens with AtomicNET on Linux to start, though we are generating the C# bindings on Linux now :)